WPFWijkagent/WPFWijkagent/MainWindowController.cs

```diff
@@ -96,6 +97,25 @@ public static List<Offence> GetOffences()
             return _offences;
         }
 
+        /// <summary>
+        /// The method executes a LINQ search on the List items and finds the offencelistItem with the same pin. 
+        /// </summary>
+        /// <returns>The method returns the offence that has the same pin</returns>
+        public static Offence RetrieveOffence(double latitude, double longitude)
+        {
+            Offence o = null;
+            IEnumerable<Offence> offenceQuerry =
+            from OffenceItem in _offences
+            where OffenceItem.LocationID.Latitude == latitude
+            && OffenceItem.LocationID.Longitude == longitude
+            select OffenceItem;
+            foreach (var item in offenceQuerry)
```

Comment:
mischien deze gewoon weg halen gezien je er zowiezo maar 1 wil retourneren. deze loop is dus overbodig

Reply:
Verwijderd